scripts/build.kit2.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const sane = require('sane');
+const { spawn } = require('child_process');
+
+const SPAWN_OPTS = { shell: true, stdio: 'inherit' };
+
+function execute(command, options = []) {
+	const resultOptions = [];
+	resultOptions.push(...options);
+
+	spawn(
+		command,
+		resultOptions,
+		SPAWN_OPTS
+	);
+}
+
+async function watchForBuild(projectFolderForWatch) {
+	return new Promise((resolve, _reject) => {
+		const dirPathForWatch = `packages/${projectFolderForWatch}`;
+		const fileForWatch = 'dist\\public-api.d.ts';
+
+		sane(dirPathForWatch)
+			.on('all', (_eventType, fileName) => {
+				if (fileName === fileForWatch) {
+					resolve();
+				}
+			});
+	});
+}
+
+async function clearDists() {
```

Comment:
please use clean script
example you can find here: https://github.com/codetheflow/skintest/blob/master/packages/cli/package.json